src/test/java/seedu/address/testutil/TypicalBuyers.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.testutil;
+
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EMAIL_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EMAIL_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PHONE_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
+
+import seedu.address.model.property.Buyer;
+
+/**
+ * A utility class containing a list of {@code Buyer} objects to be used in tests.
+ */
+public class TypicalBuyers {
+
+    // Uses TypicalPersons to build TypicalBuyers
+    public static final Buyer ALICE = new BuyerBuilder().withPerson(seedu.address.testutil.TypicalPersons.ALICE)
```

Comment:
Should we have distinct names for ```TypicalBuyer``` static final objects?
Perhaps like ```B_ALICE``` to avoid having to type out ```seedu.address.testutil.TypicalPersons.ALICE``` in case of collisions
(and also to  avoid confusion)

Reply:
Yep, included in the latest commit